dockers/docker-config-engine-stretch/Dockerfile.j2

```diff
@@ -8,11 +8,7 @@ RUN apt-get update         && \
     apt-get install -y        \
         # Dependencies for sonic-cfggen
         build-essential       \
-        python-pip            \
-        python-dev            \
-        python-setuptools
-
-RUN pip install --upgrade pip
```

Comment:
not sure why we need to modify stretch docker.

Reply:
This is simply to align the way we install/upgraded pip everywhere